plugins/GSdx/resource.h

```diff
@@ -76,70 +76,73 @@
 #define IDC_CONFIGURE                   2096
 #define IDC_COLORSPACE                  2097
 // Hardware Hacks:
-#define IDC_WILDHACK                    2100
-#define IDC_ALPHAHACK                   2101
-#define IDC_OFFSETHACK                  2102
-#define IDC_OFFSETHACK_TEXT             2103
-#define IDC_SKIPDRAWHACK                2104
-#define IDC_SKIPDRAWHACKEDIT            2105
-#define IDC_SPRITEHACK                  2106
-#define IDC_SPRITEHACK_TEXT             2107
-#define IDC_MSAACB                      2108
-#define IDC_MSAA_TEXT                   2109
-#define IDC_ALPHASTENCIL                2110
-#define IDC_TCOFFSETX                   2111
-#define IDC_TCOFFSETX2                  2112
-#define IDC_TCOFFSETY                   2113
-#define IDC_TCOFFSETY2                  2114
-#define IDC_ROUND_SPRITE                2115
-#define IDC_ROUND_SPRITE_TEXT           2116
-#define IDC_ALIGN_SPRITE                2117
-#define IDC_TC_DEPTH                    2118
-#define IDC_PRELOAD_GS                  2119
-#define IDC_FAST_TC_INV                 2120
-#define IDC_AUTO_FLUSH                  2121
-#define IDC_MEMORY_WRAPPING             2122
-#define IDC_TRI_FILTER                  2123
-#define IDC_TRI_FILTER_TEXT             2124
-#define IDC_UNSCALE_POINT_LINE          2125
-#define IDC_MERGE_PP_SPRITE             2126
+#define IDC_WILDHACK                    2098
+#define IDC_ALPHAHACK                   2099
+#define IDC_OFFSETHACK                  2100
+#define IDC_OFFSETHACK_TEXT             2101
+#define IDC_SKIPDRAWHACK                2102
+#define IDC_SKIPDRAWHACKEDIT            2103
+#define IDC_SPRITEHACK                  2104
+#define IDC_SPRITEHACK_TEXT             2105
+#define IDC_MSAACB                      2106
+#define IDC_MSAA_TEXT                   2107
+#define IDC_ALPHASTENCIL                2108
+#define IDC_TCOFFSETX                   2109
+#define IDC_TCOFFSETX2                  2110
+#define IDC_TCOFFSETY                   2111
+#define IDC_TCOFFSETY2                  2112
+#define IDC_ROUND_SPRITE                2113
+#define IDC_ROUND_SPRITE_TEXT           2114
+#define IDC_ALIGN_SPRITE                2115
+#define IDC_TC_DEPTH                    2116
+#define IDC_PRELOAD_GS                  2117
+#define IDC_FAST_TC_INV                 2118
+#define IDC_AUTO_FLUSH                  2119
+#define IDC_MEMORY_WRAPPING             2120
+#define IDC_TRI_FILTER                  2121
+#define IDC_TRI_FILTER_TEXT             2122
+#define IDC_UNSCALE_POINT_LINE          2123
+#define IDC_MERGE_PP_SPRITE             2124
 // Shader:
-#define IDC_SHADEBOOST                  2140
```

Comment:
I'm guessing the number gap was to avoid having to change the whole RC file every time something is added/removed. Dunno.